test/api/repositories/template-inputs/004-ok/expected-response.json

```diff
@@ -20,6 +20,15 @@
               "type": "string",
               "kind": "hidden",
               "default": ""
+            },
+            {
+              "id": "oauth",
+              "name": "Shopify oAuth",
+              "description": "Shopify Authorization",
+              "type": "object",
+              "kind": "oauth",
+              "default": {},
+              "componentId": "keboola.ex-shopify"
```

Comment:
Tady je E2E test. Přidal jsem oauth input a tady v `expected-response.json` je to, co očekávám že to API vrátí. A čekám že vrátí i ten `componentId`. Testy prochází zeleně, takže to funguje. 🙂